tests/integration_tests/performance/configs/test_block_performance_config_5.10.json

```diff
@@ -131,6 +131,46 @@
                                                     "delta_percentage": 7
                                                 }
                                             }
+                                        },
+                                        "syncio_1vcpu_1024mb.json": {
```

Comment:
If we use separate files, a looot of the logic found in block_performance would be duplicated.

Changing the structure of the dict is tricky since the comparison criteria is based on the structure of the config file.